### DISCUSSION TITLE
Basic question: required preprocessing

### DISCUSSION THREAD

**Discussion by giorgio-denunzio** [MIC-DKFZ/nnUNet#1009] [Q&A] [answered]
Hi all,
this is maybe too trivial but I have just started testing nnUNet on my datasets, so please excuse me.

1) At https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/dataset_conversion.md I read:
"For each training case, all images must have the same geometry to ensure that their pixel arrays are aligned. Also make sure that all your data is co-registered!"
What does this exactly mean? 
Does same geometry mean same pixel size, same slice thickness, same orientation? So should I reslice as necessary, reallocate in a similar matrix...?
Does "co-registered" mean that the target organs must be coregistered? or are you simply referring to the coregistration of masks with images (which on the other hand should be obvious so it should require no remark)?

2) What about MRI standardization? It happens that the images coming even from a single source (an hospital) have different grey-level ranges for the various organs, even if they are of the same modality: is standardization somegow taken care of in the pipeline, or should we standardize the images preliminarily?

Thanks
Giorgio

**Comment by giorgio-denunzio** [comment]
OK, I reply by myself (so to say) to question 1; I've found the answer here https://github.com/MIC-DKFZ/nnUNet/discussions/837. "Same geometry" etc refers to different modalities of the same subject. 
The second question remains, except if in the meantime I find the answer in other discussions....
Thanks!
Giorgio

**Comment by FabianIsensee** [comment, marked as answer]
MRI is normalized with z-scoring and we let nnU-Net learn to deal with the rest. Works quite well.
Best,
Fabian
